### ISSUE TITLE
build_gcc.sh does not install needed prerequisites.

### ISSUE THREAD

**Issue by sduensin** [tpoechtrager/osxcross#403]
Before `build_gcc.sh` runs `configure` it needs to run `./contrib/download_prerequisites` from the top-level GCC directory.

**Comment by jeroen** [comment]
If you run [get_dependencies.sh](https://github.com/tpoechtrager/osxcross/blob/master/tools/get_dependencies.sh) first, it seems to work?

**Comment by sduensin** [comment]
> If you run [get_dependencies.sh](https://github.com/tpoechtrager/osxcross/blob/master/tools/get_dependencies.sh) first, it seems to work?

No.  That's the dependencies for OSXCROSS itself.  (And it's even missing some things.)  `download_prerequisites` downloads several more repositories that are needed in order for GCC to be compiled.
